nixos/lib/make-loader-options.nix

```diff
@@ -0,0 +1,38 @@
+{ lib }:
```

Comment:
rename file `make-bootloader-options`, and be mindful about loader vs. bootloader usage everywhere.